posts/2023-04-27-wasi-target-rename-1.71.0.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Renaming the WASI target in Rust 1.71.0"
+author: Yoshua Wuyts
+release: false
+---
+
+In Rust 1.71 the existing `wasm32-wasi` target will be renamed to
+`wasm32-wasi-preview1`. This matches the naming used in the [WASI Preview
+Framework][wpf] and more accurately captures the evolving nature of the WASI
+targets. By making this change we also prepare Rust to re-adopt the
+`wasm32-wasi` target name for the eventual stable WASI 1.0 release.
```

Comment:
To me, this is kind of the key motivation for making this change: we eventually want `wasm32-wasi` to be the WASI 1.0 target. Not sure I have a great suggestion here but I would consider trying to reword this sentence to make that more prominent. 

I imagine many readers will end up here because their target name changed so explaining and making it obvious why this is an important enough change to break their workflow is important. 🙂

Reply:
done!